### ISSUE TITLE
This copy of libswiftCore.dylib requires an OS version prior to 12.2.0

### ISSUE THREAD

**Issue by acecilia** [google/xctestrunner#16]
Hi :)

When running version `0.2.11` of the `xctestrunner` I am getting the following error when running unit tests on swift modules:

```
================================================================================
==================== Test output for //Libraries/SwiftModule3:SwiftModule3Tests:
2020-04-28 10:48:56,633 Will consider the test as test type logic_test to run.
2020-04-28 10:48:56,990 Failed to import biplist module. Will use tool /usr/libexec/PlistBuddy to handle the binary format plist.
2020-04-28 10:48:58,542 Failed to import biplist module. Will use tool /usr/libexec/PlistBuddy to handle the binary format plist.
2020-04-28 10:48:58,550 Creating a new simulator:
Name: New-iPhone SE (2nd generation)-13.4
OS: iOS 13.4
Type: iPhone SE (2nd generation)
2020-04-28 10:48:58,787 Created new simulator EFF9153E-C6BF-4FD6-9DAF-1084F2D4F270.
This copy of libswiftCore.dylib requires an OS version prior to 12.2.0.
Child process terminated with signal 6: Abort trap
2020-04-28 10:49:08,039 Deleting simulator EFF9153E-C6BF-4FD6-9DAF-1084F2D4F270 asynchronously.
2020-04-28 10:49:08,049 Done.
```

I see this happening consistently in some scenarios:

* I saw it happening with bazel `2.2.0` and `3.0.0`. I saw it happening with Xcode `11.4` and `11.4.1`, but not with `11.3.1`. I saw it in my CI (github actions) and also running locally.
* I have seen it when running unit tests for swift modules, but not when running unit tests for iOS apps (with a host app).

Find a CI run failing due to this error here: https://github.com/acecilia/BuildSystemGenerator/actions/runs/89648161

Rolling back to version `0.2.10` seems to fix the problem.

**Comment by BORUC228** [comment]
Same trouble

**Comment by albertdai** [comment]
Fixed the issue in 0.2.12 release. Will push the release to Bazel tmr.

**Comment by albertdai** [comment]
The 0.2.12 was pushed in bazel in https://github.com/bazelbuild/rules_apple/commit/f8b83e51a8b9e83b292de5ac55d9542e5c09595f
